packages/client/components/MenuItem.tsx

```diff
@@ -112,6 +118,7 @@ const MenuItem = forwardRef((props: Props, ref: any) => {
       role='menuitem'
       ref={itemRef}
       isActive={isActive}
+      to={to}
```

Comment:
+1 Doesn't seem to be used here.

Reply:
Added some context below.

Reply:
Added some context below. This helps to support the `to` property on the `MenuItemLink` label in the sample use case below.

Reply:
Looked at this locally - we shouldn't need this here, since `to` is exposed to the parent via `useImperativeRef` and isn't used as part of `MenuItemStyles`.